CHANGELOG.md

```diff
@@ -11,15 +11,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+- Fix incorrect type definitions in date formatting.
+- Fix the Jest configuration to not use the deprecated 'tsConfig' spelling of 'tsconfig'.
 - Fix removeEventListener bug for WithTooltip.
 - Reset `isAudioOn` state to `true` when `LocalAudioOutputProvider` unmounts.
 
 ### Added
 
+- - Added some key properties to 'ModalButtonGroup', which quiets some React warnings.
```

Comment:
Nit: Could you please remove extra "-" in front of the changlog line and backtick ModalButtonGroup, 'ModalButtonGroup' to `ModalButtonGroup` ?

Reply:
Good catch. Will remove.